test/cookie.test.js

```diff
@@ -202,7 +202,7 @@ test('expires should not be overridden in clearCookie', (t) => {
 })
 
 test('parses incoming cookies', (t) => {
-  t.plan(6 + 3 * 3)
+  t.plan(6 + 3 * 2)
```

Comment:
Why is this not simply 12?

Reply:
It was written like this on the original source code, I can change it to 12 as part of this PR ^^

Reply:
Please do. It's far easier to read than an equation.